src/main/java/io/carbynestack/cli/client/castor/util/TupleFileParser.java

```diff
@@ -35,12 +37,36 @@ public static TupleChunk parse(TupleType tupleType, File tupleFile, UUID chunkId
 
     byte[] bytes;
     try (FileInputStream fileInputStream = new FileInputStream(tupleFile)) {
-      bytes = toByteArray(fileInputStream);
+      final byte[] fileBytes = toByteArray(fileInputStream);
+      bytes = removeTupleHeader(fileBytes);
```

Comment:
This adds support for Tuple files generated with MP-SPDZ v0.2.8 or later, but is thereafter no longer compatible with tuple files without headers from earlier versions. For me this seems acceptable, but should definitely be mentioned explicitly. 
@strieflin other opinions?

Reply:
@sbckr where is the right place for that - in release notes or here in code?

Reply:
I'd propose release notes and readme